modules/utk_lib_digital_custom/utk_lib_digital_custom.module

```diff
@@ -7,26 +7,65 @@
 function utk_lib_digital_custom_block_info()
 {
     $blocks['utk_lib_digital_custom_header'] = array(
-        'info' => t('Digital Collections Header'),
+        'info' => t('UTK Digital Collections Header'),
+        'cache' => DRUPAL_NO_CACHE,
+    );
+    $blocks['utk_lib_digital_custom_banner'] = array(
+        'info' => t('UTK Digital Collections Banner'),
         'cache' => DRUPAL_NO_CACHE,
     );
-
     return $blocks;
 }
 
 function utk_lib_digital_custom_block_view($delta = '')
 {
     $block = array();
-    switch ($delta) {
+    module_load_include('inc', 'islandora', 'includes/utilities');
+    $arrowmont = array("curriculum", "arrpgimg", "arrscrap", "arrsimple", "sturley", "3d");
+    $gsmrc = array("roth", "50yrcove", "colloquy", "webster", "kintner", "adams", "pcard00", "smhc", "rfj", "thompson", "wcc", "wderfilms", "derris");
+
+    $is_a_introduction = drupal_match_path(drupal_lookup_path('alias', current_path()), '*collection');
+    $parsed_request = explode('/', $_SERVER['REQUEST_URI']);
+
+    $pid = urldecode(end($parsed_request));
+    $member_of = "";
+    // If not an introduction page determine the PID
+    if (!$is_a_introduction) {
+        $object = islandora_object_load($pid);
```

Comment:
@mathewjordan This is where the last segment of the URL is used to access the Islandora object, giving the PID and other related inform.